assets/js/iframeResizer.contentWindow.js

```diff
@@ -0,0 +1,1102 @@
+/*
+ * File: iframeResizer.contentWindow.js
+ * Desc: Include this file in any page being loaded into an iframe
+ *       to force the iframe to resize to the content size.
+ * Requires: iframeResizer.js on host page.
+ * Doc: https://github.com/davidjbradshaw/iframe-resizer
+ * Author: David J. Bradshaw - dave@bradshaw.net
+ * Contributor: Jure Mav - jure.mav@gmail.com
+ * Contributor: Ian Caunce - ian@hallnet.co.uk
+ */
+
+( function( undefined ) {
+	'use strict';
+
+	if ( typeof window === 'undefined' ) {
+		return;
+	} // don't run for server side render
+
+	let
```

Comment:
Given that this is enqueued verbatim, the ES2015 `let` won't fly in IE11. I wonder if we need a huge library for this, or if we can lean on prior art like the [`Sandbox` component frame observing](https://github.com/WordPress/gutenberg/blob/a756cd8/components/sandbox/index.js#L79-L128) or [similar frame resizing in `mce-view.js`](https://github.com/WordPress/WordPress/blob/4.8.1/wp-includes/js/mce-view.js#L494-L666).

Reply:
Yeah, `let` got put it because I ran an eslint fix on these files. so the original I found is just straight es5. I think finding something smaller is a better solution though.